components/brave_wallet_ui/common/slices/api.slice.ts

```diff
@@ -162,6 +167,18 @@ export function createWalletApi (
     baseQuery: () => {
       return { data: apiProxyFetcher() }
     },
+    extractRehydrationInfo(action, { reducerPath }) {
```

Comment:
can we make sure to refresh after we see a submitted transaction is completed too?

Reply:
I configured the api slice to refetch any data that is older than 30 seconds old and/or when the app regains focus. Rehydration should only happen on app startup and is used to refresh all of the data within the slice, so it doesn't make sense to me to wipe the entire cache when a transaction updates. Instead, I have updated my PR to ensure that the cached value for a specific transaction will update when the `onTransactionStatusChanged` event is fired from the `wallet_api_proxy`. I need to complete resolve [this issue](https://github.com/brave/brave-browser/issues/28800) in order to properly update the aggregated transaction history

Reply:
Ok, that seems like a better way to handle this case. Good idea!